build/yamls/antrea-node-init.yml

```diff
@@ -0,0 +1,42 @@
+kind: DaemonSet
```

Comment:
Yes.. moved

Reply:
I disagree. This is a standalone yaml, not a kustomize patch, so IMO it's confusing to place it under "patches/gke", and it belongs under "build/yamls/gke". It will also make the link shorter, which doesn't hurt.

Reply:
@antoninbas must be right. Sorry for the confusion.